content/en/blog/_posts/2021-07-14-upcoming-changes-in-kubernetes-1-22/index.md

```diff
@@ -94,12 +97,13 @@ and explain the steps you'll need to take.
 
 `TokenReview`
 : Migrate to use the **authentication.k8s.io/v1** [TokenReview](/docs/reference/kubernetes-api/authentication-resources/token-review-v1/)
-  API, available since v1.10.
+  API, available since v1.10.  
```

Comment:
```suggestion
  API, available since v1.10. 
```
Minor nit to remove spacing (non-blocking)

Reply:
Oh that's actually intentional. 2 spaces at the end of a line in the Markdown source turns into a `<br />` in the rendered page. The other entries have that too.

Reply:
Oh! Today I learned! Thank you, @sftim 🚀

Reply:
I'm using it because paragraphs inside `<dd>` elements don't render how I want, with vertical whitespace between each paragraph (I haven't looked into why).